modules/TestUtils.js

```diff
@@ -28,6 +28,12 @@ export const isArray = (object) =>
   Array.isArray(object)
 
 /**
+ * Returns true if the given object is a set.
+ */
+export const isSet = (object) =>
+  Object.prototype.toString.call(object) === '[object Set]'
```

Comment:
So what what would you think be better? `object[Symbol.toStringTag] === 'Set'` or `object[Symbol.iterator] !== undefined`?